src/test/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProviderTest.java

```diff
@@ -103,37 +91,17 @@ public String getJobURL( Job<?, ?> project )
 
     @Test
     public void urlsWithSysPropProvider() throws Exception {
-        System.setProperty("jenkins.displayurl.provider",
+        System.setProperty(DisplayURLProvider.JENKINS_DISPLAYURL_PROVIDER_PROP,
                            TestSysPropDisplayURLProvider.class.getName());
         MockFolder folder = rule.createFolder("my folder");
-        FreeStyleProject project = (FreeStyleProject) folder
-            .createProject(rule.jenkins.getDescriptorByType(FreeStyleProject.DescriptorImpl.class), "my job",
-                           false);
-        Run<?, ?> run = project.scheduleBuild2(0).get();
-
-        String root = DisplayURLProvider.get().getRoot();
-        assertEquals("http://localhost:" + rule.getLocalPort() + "/jenkins/", root);
-        assertEquals(root + "job/my%20folder/job/my%20job/1/" + TestSysPropDisplayURLProvider.EXTRA_CONTENT_IN_URL,
-                     DisplayURLProvider.get().getRunURL(run));
```

Comment:
This test shows the exact issue described in [JENKINS-69006](https://issues.jenkins.io/browse/JENKINS-69006) - `/display/redirect` was bypassed entirely and the external URL was static.